packages/components/FocusZone/macos/RCTFocusZone.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#import <React/RCTComponent.h>
+#import <React/RCTView.h>
+
+typedef enum {
+	FocusZoneDirectionBidirectional,
+	FocusZoneDirectionHorizontal,
+	FocusZoneDirectionVertical,
+	FocusZoneDirectionNone
+} FocusZoneDirection;
+
+typedef enum {
+	NavigateStopAtEnds,
+} NavigateAtEnd;
+
+@interface RCTFocusZone : RCTView
+
+@property(nonatomic) BOOL disabled;
```

Comment:
nit : isDisabled?

Reply:
This property is exported and the name needs to match the JS API.